extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-continuous-testing.js

```diff
@@ -249,7 +278,23 @@ export class QwcContinuousTesting extends QwcHotReloadElement {
             </vaadin-grid>`;
 
         }else{
-            return html`No tests`;
+            return html`<qwc-no-data message="Continuous Testing is not running. Click the Start button or press [r] in the console to start." 
+                                    link="https://quarkus.io/guides/continuous-testing"
+                                    linkText="Read more about Continuous Testing">
+                            ${this._renderPlayButton()}
+                </qwc-no-data>
+            `
+        }
+    }
+
+    _renderPlayButton(){
+        if(!this._busy && !this._testsEnabled){
+            return html`<vaadin-button theme="tertiary" @click=${this._start}>
+                        <vaadin-icon icon="font-awesome-solid:play"></vaadin-icon>
+                        Start
+                    </vaadin-button>`;
```

Comment:
@michalvavrik - Use this button.

Reply:
thanks, but it has no ID, you can't refer to elements in shadow DOM easily by selectors based on path.

Reply:
if you are fine with adding path, your suggestion should help me